src/utils/helpers.js

```diff
@@ -50,11 +53,17 @@ function remotelyExec (machineName, zone, command, cb) {
       const trimmed = String.prototype.trim(data)
```

Comment:
Same here, `String.prototype.trim()` will call trim on an object representing an empty string. 

Need to set the string to trim as the object context to execute `trim()` on either by calling `data.trim()` or `String.prototype.trim.call(data)`